sandbox/libs/authn/src/main/java/org/opensearch/authn/AuthenticationTokenHandler.java

```diff
@@ -73,4 +81,18 @@ private static AuthenticationToken handleBasicAuth(final BasicAuthToken token) {
 
         return new UsernamePasswordToken(username, password);
     }
+
+    private static AuthenticationToken handleBearerAuth(final BearerAuthToken token) throws BadCredentialsException {
+
+        String encodedJWT = token.getHeaderValue().substring("Bearer".length()).trim();
+
+        // This just verifies the token and returns a decoded one in the process--the decoded version is not used here though
+        try {
+            JwtToken jwtToken = JwtVerifier.getVerifiedJwtToken(encodedJWT);
```

Comment:
Can we reflactor this method to be `verifyJwtToken(...)` rather that seemly misuse it?  Internally we could have `decodeJwtToken(...)` make a call to this verify method

Reply:
Yeah great call.